src/Sep.Test/ReadMeTest.cs

```diff
@@ -55,31 +56,182 @@ public void ReadMeTest_()
         // Short names and repeated constants are only for demonstration.
     }
 
+    [TestMethod]
+    public void ReadMeTest_LocalFunction_YieldReturn()
+    {
+        var text = """
+                   Key;Value
+                   A;1.1
+                   B;2.2
+                   """;
+        var expected = new (string Key, double Value)[] { ("A", 1.1), ("B", 2.2), };
+
+        using var reader = Sep.Reader().FromText(text);
+        var actual = Enumerate(reader).ToArray();
+
+        CollectionAssert.AreEqual(expected, actual);
+
+        static IEnumerable<(string Key, double Value)> Enumerate(SepReader reader)
+        {
+            foreach (var row in reader)
+            {
+                yield return (row["Key"].ToString(), row["Value"].Parse<double>());
+            }
+        }
+    }
+
+    [TestMethod]
+    public void ReadMeTest_Enumerate()
+    {
+        var text = """
+                   Key;Value
+                   A;1.1
+                   B;2.2
+                   """;
+        var expected = new (string Key, double Value)[] { ("A", 1.1), ("B", 2.2), };
```

Comment:
```suggestion
        var expected = new (string Key, double Value)[] {
            ("A", 1.1),
            ("B", 2.2),
        };
```

Reply:
i think it would be more easy to spot the connection if expected was more similar to `text`